src/link/seriallink.cpp

```diff
@@ -6,6 +6,16 @@
 
 #include "seriallink.h"
 
+#ifdef Q_OS_LINUX
+    #include <sys/ioctl.h>
+    #include <termios.h>
+    #include <linux/serial.h>
+#endif
+#ifdef Q_OS_WIN
+    #include <Windows.h>
```

Comment:
windows.h - non capitals. Windows is moving torwards to a case sensitive file system in the future.

Reply:
From QtCreator:
![image](https://user-images.githubusercontent.com/1215497/63035475-96bcc600-be91-11e9-9bdf-e2f65818a6d2.png)

Should we use `Windows` or `windows` ?

Reply:
then  I got it wrong and the case of the Windows.h is actually in capitals.